src/handlers/connect/shell-connect.handler.ts

```
@@ -75,20 +86,7 @@ export async function connectHandler(
 
     // make a new connection
     const connectionHttpService = new ConnectionHttpService(configService, logger);
-    // if SSM user does not exist then resp.connectionId will throw a
-    // 'TypeError: Cannot read property 'connectionId' of undefined'
-    // so we need to catch and return undefined
-    const connectionId = await connectionHttpService.CreateConnection(parsedTarget.type, parsedTarget.id, cliSpaceId, targetUser).catch(() => undefined);
-
-    if(! connectionId)
-    {
-        logger.error('Connection creation failed');
-
-        logger.error(`You may not have a policy for targetUser ${parsedTarget.user} in environment ${parsedTarget.envName}`);
-        logger.info('You can find SSM user policies in the web app');
-
-        await cleanExit(1, logger);
-    }
```

Why isn't this needed anymore?

---

This code was previously catching an error that occurs in `CreateConnection` and returning undefined:

```
// 'TypeError: Cannot read property 'connectionId' of undefined'
// so we need to catch and return undefined
const connectionId = await connectionHttpService.CreateConnection(parsedTarget.type, parsedTarget.id, cliSpaceId, targetUser).catch(() => undefined);    
```

I think this code was from a very long time ago and was just suppressing the actual error that occurred. Now if an error occurs in the `connectionHttpService` it should be treated as a fatal error for that `zli connect` command and the error should just be caught at the top level by yargs and printed to the user. The code was also assuming an error could happen because of policy but that also doesnt really apply anymore since most policy errors should get caught [here](https://github.com/bastionzero/zli/blob/755af9ef2f093401a1ca351cbbea7c4ed24cf1ba/src/handlers/connect/shell-connect.handler.ts#L49-L74) before the `CreateConnection` call even happens.